data/sql/updates/pending_db_world/rev_1592650543169399600.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1592650543169399600');
+
+UPDATE `smart_scripts` SET `link` = 0 WHERE `entryorguid` IN (16398,8338,178908,32347,24077) AND `id` = 0;
+UPDATE `smart_scripts` SET `link` = 0 WHERE `entryorguid` = 30895 AND `id` = 1;
```

Comment:
just FYI, you could combine them into one single `UPDATE` statement as they do the same thing:

```sql
UPDATE `smart_scripts` SET `link` = 0
WHERE (`entryorguid` IN (16398,8338,178908,32347,24077) AND `id` = 0);
OR (`entryorguid` = 30895 AND `id` = 1);
```

I haven't tested it, but the result should be the same

Reply:
That's one hella query xD gonna test